packages/framework-integration-tests/integration/provider-unaware/load/read-models-are-correctly-projected.load.ts

```diff
@@ -0,0 +1,54 @@
+import { ApolloClient } from 'apollo-client'
+import { NormalizedCacheObject } from 'apollo-cache-inmemory'
+import { internet, random } from 'faker'
+import { expect } from '../../helper/expect'
+import gql from 'graphql-tag'
+
+import { waitForIt } from '../../helper/sleep'
+import { applicationUnderTest, scriptExecutor } from './setup'
+
+describe('Data consistency on read models', () => {
```

Comment:
This is the second load test. It is focused mainly on data corruption of read models when just one instance aggregates a lot of entities. This exercises the optimistic concurrency